chronos.nimble

```diff
@@ -9,26 +9,45 @@ skipDirs      = @["tests"]
 
 requires "nim > 0.18.0"
 
+import ospaths
+
 task test, "Run all tests":
-  for tfile in @[
-      "testsync",
-      "testsoon",
-      "testtime",
-      "testfut",
-      "testsignal",
-      "testaddress",
-      "testdatagram",
-      "teststream",
-      "testserver",
-      "testbugs",
-    ]:
-    for cmd in @[
-        "nim c -r -d:useSysAssert -d:useGcAssert tests/" & tfile,
-        "nim c -r tests/" & tfile,
-        #"nim c -r --gc:markAndSweep tests/" & tfile,
-        "nim c -r -d:release tests/" & tfile
-      ]:
-      echo "\n" & cmd
-      exec cmd
-      rmFile("tests/" & tfile.toExe())
 
+  var testFiles = @[
```

Comment:
by the way, in the other projects we have a single `all_tests` which just imports each test... it's pretty nice because it doesn't have to compile each test separately (but you can still do it if you want to, on command line for example..)